src/main/java/plugily/projects/buildbattle/arena/ArenaRegistry.java

```diff
@@ -61,9 +61,6 @@ public PluginArena getNewArena(String id) {
       case "guessthebuild":
       case "guess_the_build":
         return new GuessArena(id);
-      case "classic":
-      case "solo":
-      case "team":
```

Comment:
These was removed because it is pretty useless to keep as there is a `default` specified under this.

Reply:
yeah it was there as if another mode would come it doesn't need to get typed but its also okay to remove, Idc